types/victor/index.d.ts

```diff
@@ -337,8 +337,9 @@ declare class Victor {
 
     /**
      * Rounds both axis to a certain precision.
+     * @param {Number} precision (default: 8)
```

Comment:
The type hint probably shouldn't be here since it's already provided by TypeScript

```suggestion
     * @param precision (default: 8)
```

Reply:
Hey @MysteryBlokHed that's a good point I just ammended my commit to remove the hint 👍